CHANGELOG.md

```diff
@@ -3,6 +3,23 @@ Changelog for NeoFS LOCODE database
 
 ## [Unreleased]
 
+## [0.7.0] - 2024-08-28
+
+### Changed
+- Go 1.22 is required to build now (#40, #43)
+- Added a file with which you can overwrite the coordinates of incorrect 
+records (#38)
```

Comment:
Usually lines are not breaked here, if they are then you're missing two leading spaces on this line.